### PR TITLE
DicomAttribute not generated in XML when element is of type DicomFragmentSequence

### PR DESCRIPTION
**Pull request by ghost**
**DicomAttribute not generated in XML when element is of type DicomFragmentSequence**

Fixes # .

#### Checklist
- [x] The pull request branch is in sync with latest commit on the *fo-dicom/development* branch
- [ ] I have updated API documentation
- [x] I have included unit tests
- [x] I have updated the change log
- [x] I am  listed in the CONTRIBUTORS file

#### Changes proposed in this pull request:
-Changes made in DiocmXML class to handle DicomFragmentSequence
-Signature of GetBinaryBase64 changed to make it generic for dicom elements
-New file added to test data for testing
-Unit test added to make sure new changes are working expected

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -199,3 +199,10 @@
 * in update to DICOM2020e the keywords, that are provided by Nema, are used. therefore some DicomUID-Names changed.
 
 
+#### 5.1.1 (2023-08-27)
+
+##### Changes:
+* Fix the issue of 'DicomAttribute not generated in XML when element is of type DicomFragmentSequence'
```
Comment (gofal) [inline ChangeLog.md]: These listed changes should be added on the top of the file within the section "### 5.1.2 (TBD)"

File: Tests/FO-DICOM.Tests/Serialization/XmlDicomConverterTest.cs [hunk 2]
```diff
@@ -62,5 +64,27 @@ public void TestSequenceXmlSerialization()
...
+
+        [Fact]
+        public void TestDicomFragmentSequenceXmlSerialization()
+        {
+            // Test File HasFragmentSequence with DicomFragmentSequence (total count is 96)
+            var file = DicomFile.Open(TestData.Resolve("HasFragmentSequence.dcm"));
+            var xml = DicomXML.ConvertDicomToXML(file.Dataset);
+
+            var PixelDataStart = new StringBuilder();
+            PixelDataStart.AppendLine(@"<DicomAttribute tag=""7FE00010"" vr=""OB"" keyword=""PixelData"">");
```
Comment (gofal) [inline Tests/FO-DICOM.Tests/Serialization/XmlDicomConverterTest.cs]: "pixelDataStart" could just be a string constant, no need for a StringBuilder class here

**Review by gofal — CHANGES_REQUESTED** [review]
Great, thanks!

File: ChangeLog.md (hunk 1 above)
Comment (ghost) [reply ChangeLog.md]: Sure

File: Tests/FO-DICOM.Tests/Serialization/XmlDicomConverterTest.cs (hunk 2 above)
Comment (ghost) [reply Tests/FO-DICOM.Tests/Serialization/XmlDicomConverterTest.cs]: Sure

Comment (ghost) [reply Tests/FO-DICOM.Tests/Serialization/XmlDicomConverterTest.cs]: Hey did you get a chance to review the changes.

**Review by gofal — APPROVED** [review]

**Pull request merged**